### ISSUE TITLE
Cannot detach database named local

### ISSUE THREAD

**Issue by jeffmotherduck** [duckdb/duckdb#8282]
### What happens?

If you try to open a database named "local.db", it creates a database with the name local. If you then try to attach local, you cannot detach it because local is a reserved word.

### To Reproduce

```
D .open load.db;
D show databases;
```

```
D attach 'local.db';
D detach local;
```

### OS:

Mac OS

### DuckDB Version:

v0.8.1

### DuckDB Client:

CLI

### Full Name:

Jeff Huang

### Affiliation:

Motherduck

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Alex-Monahan** [comment]
Howdy Jeff!

In the case of reserved words, wrapping it in double quotes fixes the issue! I tested it on the 0.8.1 CLI and `detach "local"` does the trick!

**Comment by jeffmotherduck** [comment]
thanks alex! forgot about double quotes..i'll just close this then
